File: Oregon 9-1-1/Oregon 9-1-1.Android.NativeActivity/callLogEntry.cpp
#include "Oregon911.h"

callLogEntry::callLogEntry(unsigned int _id, timestamp _tc, std::string _message)
{

}

callLogEntry::callLogEntry() {

}

callLogEntry::~callLogEntry()
{

}

std::string callLogEntry::getMessage()
{
	return std::string();
}

timestamp callLogEntry::getTimestamp()
{
	return timestamp();
}

void callLogEntry::setid(unsigned int _id)
{
}

void callLogEntry::setMessage(std::string _message)
{
}

void callLogEntry::setTimestamp(timestamp _ts)
{
}

File: Oregon9-1-1/app/src/main/java/cad/oregon911/net/callLogEntry.java
package cad.oregon911.net;

/**
 * Created by Brandan on 12/28/2015.
 */
public class callLogEntry {

    private int id;
    private int userid;
    private String username;
    private timestamp ts;
    private String message;

    public callLogEntry() {
        this.id = 0;
        ts = new timestamp();
    }

    public callLogEntry(int id, timestamp ts, int userid, String username, String message) {
        this.id = id;
        this.ts = ts;
        this.userid = userid;
        this.username = username;
        this.message = message;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public timestamp getTs() {
        return ts;
    }

    public void setTs(timestamp ts) {
        this.ts = ts;
    }

    public int getUserid() {
        return userid;
    }

    public void setUserid(int userid) {
        this.userid = userid;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}
